views/add-event.hbs

```diff
@@ -0,0 +1,66 @@
+<div class="add-eventContainer">
+  <h3 class="userAddEvent">{user name} add event</h3>
+
+  <form class="add-event-form" action="/add-event" method="post">
+    <label>Title: </label>
+    <input class="form-control" type="text" name="name" placeholder="title" required>
+
+    <label>Picture: </label>
+    <input class="form-control" type="text" name="imageUrl" placeholder="choose picture">
```

Comment:
the label should and placeholder be Image URL, or at least suggest it should be a URL